readme.md

```diff
@@ -152,11 +158,11 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Question:** What is `req.params`? What else will you need to change if you change the path to `/my/name/is/:username`?
 
-* **Your Answer:**
+* **Your Answer:** these are the query parameters used in the queries url string. You will need to change the response to utilize req.params. You can change the ':username' part of the parth.
```

Comment:
In this case, we'd actually talk about this as "route parameters." "Query parameters" are what comes _after_ the route ends (i.e. after the `?`).